### ISSUE TITLE
Adding Book Name to the Title - SEO

### ISSUE THREAD

**Issue by venkat0304** [BookStackApp/BookStack#2041]
Hi, Is there any way to add Book title name to the title meta tag ?

Expecting something like <Bookname> - <Pagetitle> | <websitename>

base.blade.php
```
<!DOCTYPE html>
<html lang="{{ config('app.lang') }}" class="@yield('body-class')">
<head>
    <title>{{ isset($pageTitle) ? $pageTitle . ' | ' : '' }}{{ setting('app-name') }}</title>
    <!-- Meta -->

```

**Comment by venkat0304** [comment]
{{ $book->name }} -> adding this works on books page but shelves page is giving 500 error.

**Comment by LouWii** [comment]
Have you tried something like `{{ isset($book) ? $book->name : '' }}`

**Comment by ssddanbrown** [comment]
Hi @venkat0304,
This is not possible without altering the core BookStack files.

If you do want to make changes to the core files, using the code @LouWii has advised should work to get the book name if it exists. Alternatively you could shorten this to `{{ $book->name ?? '' }}``.
As another option, This is where the title for a page is set:

https://github.com/BookStackApp/BookStack/blob/79a949836b93c89b5efd9d9cf94e0f2b93dda48f/app/Http/Controllers/PageController.php#L148

Since this request is already open under #1604, With #1682 being very much related, I'm going to close this off as a duplicate.
